inc/notificationtargetentity.class.php

```diff
@@ -43,6 +43,8 @@ public function addDataForTemplate($event, $options = []) {
       $this->data['##credit.name##'] = $this->obj->getField('name');
       $this->data['##credit.quantity_sold##'] = $this->obj->getField('quantity');
       $this->data['##credit.enddate##'] = $this->obj->getField('end_date');
+      $this->data['##credit.overconsumption_allowed##'] = $this->obj->getField('overconsumption_allowed') ? __('Yes') : __('No');
+      $this->data['##credit.child_entities##'] = $this->obj->getField('is_recursive') ? __('Yes') : __('No');
```

Comment:
We can probably use `Dropdown::getYesNo(...)` which already do the same :

![image](https://github.com/pluginsGLPI/credit/assets/42734840/8ade38f3-b6cf-4d4a-8eca-c6c344470835)